scripts/tests/test_zkai_attention_kv_quantized_softmax_receipt_gate.py

```
@@ -0,0 +1,95 @@
+import copy
+import json
+import tempfile
+import unittest
+
+from scripts import zkai_attention_kv_quantized_softmax_receipt_gate as gate
+
+
+class QuantizedSoftmaxReceiptGateTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.12)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)